modules/custom/atlas/atlas_statistics/atlas_statistics.inc

```diff
@@ -64,18 +64,26 @@ function atlas_statistics_atlas_statistics() {
       if (in_array('site_owner', $user->roles)) {
         $usernames_site_contact[] = $user->name;
       }
+      if (in_array('content_editor', $user->roles)) {
+        $usernames_content_editor[] = $user->name;
+      }
     }
     if ($user->mail) {
       $useremails[] = $user->mail;
       if (in_array('site_owner', $user->roles)) {
         $useremails_site_contact[] = $user->mail;
       }
+      if (in_array('content_editor', $user->roles)) {
+        $useremails_content_editor[] = $user->mail;
+      }
     }
   }
   $statistics_data['users']['username']['all'] = $usernames;
   $statistics_data['users']['username']['site_contact'] = $usernames_site_contact;
+  $statistics_data['users']['username']['content_editor'] = $usernames_content_editor;
   $statistics_data['users']['email_address']['all'] = $useremails;
```

Comment:
In https://github.com/CuBoulder/atlas/pull/310, we switch from `all`  to `content_editor`. Here we are adding, which way is preferred?